mypyc/irbuild/mapper.py

```diff
@@ -145,7 +145,7 @@ def fdef_to_sig(self, fdef: FuncDef) -> FuncSignature:
         # deserialized FuncDef that lacks arguments. We won't ever
```

Comment:
This comment needs updating

Reply:
Fixed; as an extra comment-only change. I can squash all commits into one if preferred.

Thank you for taking the time to look at this @hauntsaninja !